src/plugin-notification/window/notificationmodule.cpp

```diff
@@ -52,104 +51,70 @@ NotificationModule::NotificationModule(QObject *parent)
     }
     m_model = new NotificationModel(this);
     m_worker = new NotificationWorker(m_model, this);
-
-    connect(m_model, &NotificationModel::appListAdded, this, &NotificationModule::onAppListAdded);
-    connect(m_model, &NotificationModel::appListRemoved, this, &NotificationModule::onAppListRemoved);
-}
-
-NotificationModule::~NotificationModule()
-{
-    m_worker->deleteLater();
-    m_model->deleteLater();
+    connect(m_model, &NotificationModel::appListAdded, this, &NotificationHModule::onAppListAdded);
+    connect(m_model,
+            &NotificationModel::appListRemoved,
+            this,
+            &NotificationHModule::onAppListRemoved);
```

Comment:
个人觉得这样还不如

```suggestion
    connect(m_model, &NotificationModel::appListRemoved,
            this, &NotificationHModule::onAppListRemoved);
```

相比可读一些。当然这个改不改都行。

Reply:
嗯，不听clang-format 的会好点